addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -340,10 +364,72 @@ const Wysiwyg = Widget.extend({
     redo: function () {
         this.odooEditor.historyRedo();
     },
+    /**
+     * Open the Alt dialog
+     * dialog give access to edit <img> alt and title
+     */
+    openAltDialog(params) {
+        let $baseNode = params.node ? params.node : this.lastTargetedElement;
+
+        let altDialog = new weWidgets.AltDialog(this, params, $baseNode);
+        altDialog.open();
+    },
+    /**
+     * Open the media dialog
+     * use to insert img / icons / videos
```

Comment:
To complete the documentation, I would write:
```
Open the media dialog.

Used to insert or change image, icon, document and video.
```

Reply:
How about this instead?
```js
/**
 * Open the Media dialog to insert or replace an image, an icon, a document or a video.
 * 
 * @param {object} params
 * @param {Node} [params.node]
 */
```
I don't know what that `params` object is besides its optional `node` property but apparently it's mandatory :shrug: